### ISSUE TITLE
Constant alert color on resume from suspend (gnome 3.28.2)

### ISSUE THREAD

**Issue by tnilzon** [hackedbellini/gnome-shell-notifications-alert#41]
There is a 'Constant alert color on resume from suspend' on gnome 3.28.2. Only goes back to normal on a shell restart or when there is a new notification. Please fix, it's a great extension!

**Comment by hackedbellini** [comment]
@tnilzon thanks for liking my extension :)

hrm, I've seem this issue before, but the problem is that I cannot reproduce it in my system. For example, I just tried to display a notification, locked the screen while it was blinking, returned from it to find it blinking but it did go away when I dismissed the notification.

Can you describe the issue in more details so I can try to reproduce it?

**Comment by tnilzon** [comment]
I am using Manjaro Gnome 17.1.12 (with Gnome 3.28.2). No propitiatory graphic drivers in use.

After resume from (auto-) suspend the color of the Date/Clock is what is set as alert color (e.g. red) with no flashing. To get it back to normal you have to do Alt+F2 and -r, or logout/login.

After unlocking the desktop from (auto-) screen lock the behavior condition is the same as above.

**Comment by hackedbellini** [comment]
@tnilzon yeah, that's a problem very hard to track indeed.

To be very hones, I really need to rewrite some of the code from this extension. When i first wrote it back in 2012, gjs didn't even support es6, and I also didn't know js enough to do a "great" job hahaha.

Maybe by rewriting the core of it I can make things like this go away. I'll try to do that ASAP

**Comment by tnilzon** [comment]
Yeah, that sound great! Really would appreciate if you do, I find this extension very useful. So thank's in advance, and hope you can solve it.
